DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupCourseRemove.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@inject IConfiguration Configuration;
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model GroupCourseRemoveViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = $"{(errorHasOccurred ? "Error: " : "")}Groups - Remove group course";
+
+  var routeData = new Dictionary<string, string> {
+    { "groupId", ViewContext.RouteData.Values["groupId"].ToString()! }
+  };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm), nameof(Model.DeleteStartedEnrolments) })" />
+    }
+
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-8">Are you sure you would like to remove @Model.CourseName course from this group?</h1>

```

Comment:
I don't think we should have `course` after @Model.CourseName - in many places it's unlikely to make sense.
E.g. "Are you sure you would like to remove SDF Online - Implementation and Business Change Support Manager - 2019-20 course from this group?"
We could say '...remove the course "@Model.CourseName" from this group?'. That might be more watertight.

Reply:
Yes, that reads better. Done. Trivial enough to not require clarification of change from HEE?

Reply:
I reckon so aye.
=